src/metabase/driver/bigquery.clj

```diff
@@ -339,7 +339,7 @@
 (defn- field->identifier [{table-id :table_id, :as field}]
   (let [db-id   (db/select-one-field :db_id 'Table :id table-id)
         dataset (:dataset-id (db/select-one-field :details Database, :id db-id))]
-    (hsql/raw field)))
+    (hsql/raw (get (field/qualified-name-components field) 1))))
```

Comment:
Do we want to use just the field for legacy and standard? This needs to be double checked.